source/contributing/implementation/codestyle.rst

```diff
@@ -0,0 +1,153 @@
+==========
+Code Style
+==========
+
+We follow `Google's Java Style Guidelines <https://google-styleguide.googlecode.com/svn/trunk/javaguide.html>`_ with a
```

Comment:
Style guide has moved to https://google.github.io/styleguide/javaguide.html, better to link there instead of to a page that redirects there.